app/models/master_file.rb

```diff
@@ -640,7 +643,9 @@ def get_ffmpeg_frame_data frame_source, new_width, new_height
         end
         data
       ensure
-        File.unlink(file_source) unless file_source =~ %r(https?://)
+        File.unlink file_source unless file_source =~ %r(https?://)
+        File.unlink frame_source[:source] unless frame_source[:source] =~ %r(https?://)
+        File.unlink jpeg
```

Comment:
Odd that this is necessary and that it isn't automatically being cleaned up.

Reply:
It can take many days to clean up, could fill up the drive when thousands of items are being ingested.